source/SnapDump-Core/SDMemorySnapshot.class.st

```diff
@@ -48,13 +48,15 @@ SDMemorySnapshot >> serializeContext: aContext on: stream [
 SDMemorySnapshot >> setContext: aContext exceptionClassName: exceptionClassName [
 	| method |
 	context := aContext.
+	self
+		stackTrace:
+			(String streamContents: [ :stream | context errorReportOn: stream ]).
```

Comment:
I wonder if we should not restrict the stack trace and fuel to include only a certain amount of stack information.

Reply:
What do you think should not be part of the stack or the context ?

Reply:
I'm not sure. I just had often the case that the stack got too big to report although I'm interested only in the first 50 stack frames or such. But I have nothing concrete in mind. And the textual stack trace grows slower than the fuel dump so let's think  about it for a while